src/generate.rs

```diff
@@ -63,6 +70,10 @@ pub struct GenerateArgs {
     /// Seed to initialize RNG for reproducability
     #[clap(long)]
     pub seed: Option<u64>,
+
+    /// The number of sequence pairs to generate
```

Comment:
Update this

Reply:
maybe just delete?  `pattern_length` is pretty self-explanatory

Reply:
you can explain what the pattern is / is used for